taskcluster/tc-android-ds-tests.sh

```diff
@@ -30,4 +30,6 @@ android_setup_ndk_data
 
 run_tflite_basic_inference_tests
 
+run_hotword_tests
```

Comment:
this one should call `run_hotword_android_tests`

Reply:
fixed